pkgs/api-events/src/types/Event.ts

```diff
@@ -27,8 +27,16 @@ export interface Event {
    */
   name: string;
 
+
+  /**
+   * Attributes that describe the event. 
+   * Intended to be used by instrumentation libraries.
+   */
+  data?: Attributes;
```

Comment:
I think we need to first add a new type that will support nested attributes, something like the LogAttributes interface that @Abinet18 has in https://github.com/open-telemetry/opentelemetry-sandbox-web-js/pull/112.